### DISCUSSION TITLE
Uninstallation of exe files inside the wix bootstrapper bundle

### DISCUSSION THREAD

**Discussion by itssaikiran** [wixtoolset/issues#7506] [Questions]
I created a one wix bootstrapper bundle which include 2 msi & 2 exe files. While running the bundle it is installing all those 4 packages but while doing uninstall it is not uninstalling exe files. Can anyone pls help me to sort out the issue?

Those exe files are Microsoft monitoring agent & Microsoft edge webview2 runtime.

 <ExePackage SourceFile="..\..\WIX\scripts\Configure-MMA.exe" LogPathVariable="log_var" Compressed="yes" Vital="yes"  DisplayName="Microsoft Monitoring Agent" Description="Configure-MMA" />

<ExePackage SourceFile="..\..\WIX\installers\MicrosoftEdgeWebview2Setup.exe" LogPathVariable="log_var" Compressed="yes"  Vital="yes" DisplayName="msedgeWebview2" Description="Microsoft Edge WebView2" />

**Comment by BMurri** [comment]
Did you provide uninstall command-lines for the two exe packages in your bundle?

**Reply by itssaikiran** [reply]
No. how to mention that one. what command to be used in uninstall command. can u pls suggest
